packages/newspringchurchapp/src/prayer/PrayerActionMenuCard/PrayerActionMenuCardConnected.js

```diff
@@ -3,13 +3,27 @@ import PropTypes from 'prop-types';
 import { Mutation } from 'react-apollo';
 
 import savePrayerRequest from '../data/mutations/savePrayerRequest';
+import savedPrayerList from '../data/queries/getSavedPrayers';
 import PrayerActionMenuCard from './PrayerActionMenuCard';
 
 const PrayerActionMenuCardConnected = memo(
-  ({ onAdvancePrayer, navigation, prayerId, ...props }) => {
+  ({ onAdvancePrayer, navigation, prayerId, prayerRequest, ...props }) => {
     const exitPrayer = () => navigation.navigate('Prayer');
     return (
-      <Mutation mutation={savePrayerRequest}>
+      <Mutation
+        mutation={savePrayerRequest}
+        update={async (cache) => {
+          const currentSavedPrayers = cache.readQuery({
+            query: savedPrayerList,
```

Comment:
```suggestion
            query: GET_SAVED_PRAYERS,
```

Reply:
Holding off on this until Vincent says his peace